flowetl/flowetl/flowetl/operators/staging/mount_event_operator_factory.py

```diff
@@ -0,0 +1,48 @@
+from typing import Literal
+
+from airflow.operators.postgres_operator import PostgresOperator
+from flowetl.operators.staging.event_columns import event_column_mappings
+
+
+def create_mount_event_operator(
```

Comment:
I'd prefer it if this was an operator class with a parameterised sql string rather than a factory function.

I.e. use `{{ params.event_type }}` in the sql. Could probably define a macro to handle the columns (i.e. `{{ columns_string(params.event_type) }}`).